Author: Serge Hallyn <serge.hallyn@canonical.com>
Description: include stdint.h to provide a definition for
   uint64_t, which is now used in linux/fs.h.
Forwarded: no
Index: ubuntu-libvirt/src/lxc/lxc_container.c
===================================================================
--- ubuntu-libvirt.orig/src/lxc/lxc_container.c
+++ ubuntu-libvirt/src/lxc/lxc_container.c
@@ -43,6 +43,7 @@
 #include <linux/unistd.h>
 
 /* For MS_MOVE */
+#include <stdint.h>
 #include <linux/fs.h>
 
 #if WITH_CAPNG
